src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -54,7 +49,6 @@ public Command setTeamColor(Integer... banks) {
         return new Command("Set team color", this) {
 
             protected void initialize() {
-                // Called just before this Command runs the first time
                 Color color;
 
                 if (isBlueTeam()) {
```

Comment:
Why does it determine color if it sets it to `Color.OFF` in the loop?

Reply:
It does a good?